README.md

```diff
@@ -1,6 +1,19 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Create two .env files in the root directory:
+
+.env.development
+.env.test
+
+Add the following content to their respective file:
```

Comment:
The idea behind using the .env files and gitignoring them is so that we don't have our database names here - we should instead tell users how to find the database names (in setup.sql)
(In reality, we also would not push up setup.sql as it exposes the database names too but in this instance we can't avoid it as users will need it here!)